packages/control-input/src/ControlGroup.stories.tsx

```diff
@@ -34,21 +34,27 @@ CheckboxGroupInline.args = {
 	...checkboxDefaultArgs,
 	block: false,
 };
-CheckboxGroupInline.storyName = 'Checkbox group inline';
+CheckboxGroupInline.storyName = 'Checkbox Group Inline';
```

Comment:
These stories had inconsistent naming conventions so I've fixed this while I'm here.